css/styles.css

```diff
@@ -0,0 +1,197 @@
+}
+
+main {
+  background: url(../images/your_verse_hero.jpg) center -48.55em no-repeat;
+  background-size: 1440px 1315px;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+  max-width: 100%;
+}
+
+.header {
+  width: 960px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.nav-menu {
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(../images/menu.png) 50% 50% no-repeat, -webkit-gradient(linear, 0 100%, 0 0, from(rgb(102, 102, 102)), color-stop(0.5, rgb(94, 94, 94)), color-stop(0.51, rgb(112, 112, 112)), to(rgb(128, 128, 128)));
+  border-radius: 4px;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  position: absolute;
+  top: 52px;
+  font-size: small;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.menu li {
+  float: left;
+  max-width: 100%;
+}
+
+.hero-text a {
+  background: url(../images/your_verse.png) top no-repeat;
+  display: block;
+  height: 30px;
+  text-indent: -9999px;
+}
+
+.logo a {
+  background-image: url(../images/apple-logo.svg);
+  background-position: top;
+  text-indent: -99909px;
+  width: 100px;
+}
+
+.menu li a {
+  color: #fff;
+  display: block;
+  height: 36px;
+  margin: 0;
+  text-decoration: none;
+  padding: 8px 35px;
+  max-width: 100px;
+  border-style: solid;
+  border-image: url(../images/menu1.png) 0 2 0 1;
+  border-width: 0 2px 0 1px;
+}
+
+.search {
+  margin: 8px 20px 0 20px;
+}
+
+.search input {
+  background: url(../images/icon-search.svg) no-repeat;
+  background-position: 4px 5px;
+}
+
+.search-input {
+  width: 100px;
+  height: 20px;
+  border-radius: 8px;
+  border: none;
+  color: #fff;
+  box-shadow: rgba(0, 0, 0, 0.4) 0 -1px 0, rgba(255, 255, 255, 0.2) 0 1px 0;
+  background-color: rgba(255, 255, 255, 0.1);
+}
+
+.heros {
+  background: url(../images/your_verse_hero.jpg) center bottom no-repeat;
+  background-size: cover;
+  bottom: 0;
+  left: 0;
+  max-height: 600px;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.hero-text {
+  color: #fff;
+  margin: 170px auto 140px auto;
+  max-width: 100%;
+  width: 50%;
+  text-align: center;
+  text-decoration: none;
+}
+
+.hero-text h1 {
+  background: url(../images/what_will_your_verse_be.png) no-repeat;
+  background-size: cover;
+  height: 5rem;
+  text-indent: -9999px;
+}
+
+.hero-text h2 {
+  background: url(../images/everyone_has_something_to_share.png) center top no-repeat;
+  background-size: 56.9rem;
+  height: 82px;
+  text-indent: -9999px;
+}
+
+.bottom-footer li {
+  display: inline-block;
+  width: 356px;
```

Comment:
Use more responsive units here, like em, rem or percentage, if you do this and remove the width on the body, the horizontal bar should go off